packages/td-tools/src/thing-model-helpers.ts

```diff
@@ -376,7 +376,7 @@ export class ThingModelHelpers {
         if (!options) {
             options = {} as CompositionOptions;
         }
-        if (!options.baseUrl) {
+        if (options.baseUrl === undefined) {
             options.baseUrl = ".";
         }
```

Comment:
Statements like these might actually be able to be simplified like this:

```suggestion
        options.baseUrl ??= ".";
```

Reply:
You are right, but to be honest I still have *issues* understanding the nullish coalescing operator at the first look. Is it just me? :-)
Shall we go over and update all of them?

Reply:
Is this conversation resolved? Are we going to enable the rule for eslint? if that's so we should have a tracking issue.

Reply:
I just opened #1092, so this conversation can be resolved :)